src/styles/blocks/button.scss

```diff
@@ -0,0 +1,43 @@
+.button {
+  box-sizing: border-box;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+  height: 56px;
+  border: 2px solid $color-blue;
+  border-radius: 8px;
+
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 14px;
+  text-decoration: none;
+  color: $color-blue;
+
+  transition: color 300ms,
+    box-shadow 300ms,
+    background-color 300ms,
+    border-color 300ms;
```

Comment:
you can make a variable for the duration. imagine in the future you decide to change the animation duration on your website from 300ms to 500ms. if you don't have a variable for the duration you'd need to change it everywhere and hope you didn't miss anything